test.js

```diff
@@ -3,19 +3,17 @@ var test = require('ava');
 var isOnline = require('./');
 
 test(function (t) {
```

Comment:
Use async functions to test the code. (AVA transpiles tests using Babel)

Example: https://github.com/sindresorhus/got/blob/master/test/http.js

You can also then drop the `t.plan` assertions as they're no longer useful.